test/jdk/javax/security/auth/Destroyable/KeyDestructionTest.java

```diff
@@ -42,7 +42,7 @@
 public class KeyDestructionTest {
     public static void main(String[] args) throws Exception {
         String kpgAlgorithm = "RSA";
-        KeyPair keypair = generateKeyPair(kpgAlgorithm, 1024);
+        KeyPair keypair = generateKeyPair(kpgAlgorithm, 2048);
```

Comment:
This also changes the non FIPS testing. Is there sufficient coverage for keys smaller than 2048 such that this won't allow any regression?